langchain4j-couchbase/src/test/java/dev/langchain4j/store/embedding/couchbase/CouchbaseEmbeddingStoreIT.java

```diff
@@ -0,0 +1,90 @@
+package dev.langchain4j.store.embedding.couchbase;
+
+import com.couchbase.client.java.Bucket;
+import com.couchbase.client.java.Cluster;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.AllMiniLmL6V2QuantizedEmbeddingModel;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import dev.langchain4j.store.embedding.EmbeddingStoreIT;
+import dev.langchain4j.store.embedding.EmbeddingStoreWithFilteringIT;
+import lombok.SneakyThrows;
+import org.testcontainers.couchbase.BucketDefinition;
+import org.testcontainers.couchbase.CouchbaseContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+
+@Testcontainers
+class CouchbaseEmbeddingStoreIT extends EmbeddingStoreIT {
```

Comment:
```suggestion
class CouchbaseEmbeddingStoreIT extends EmbeddingStoreWithFilteringIT {
```

Reply:
I ran into issues with filtering so, for now we decided to not implement it and release only storing and removing embeddings.